tests/e2e/features/artifacts/transferPage/atrifacts-transfers.feature

```diff
@@ -147,3 +147,40 @@ Feature: Artifacts - UI
     Then Element with "text" "ZZ" should be "visible"
     Then Element with "text" "0x1ab721...184" should be "visible"
     Then Element with "partial src" "zz.png" should be "visible"
+
+  @id1364 @Transfer
```

Comment:
do we have extra steps below or something missed in the documentation? 
![image](https://github.com/matter-labs/dapp-portal/assets/51121511/7fd13bd1-bf5e-4576-a49c-cd23252bcf2a)
Please align steps in doc and here in the same order.

Reply:
Fixed in latest commit